xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -335,21 +335,28 @@ void CRendererBase::DeleteRenderBuffer(int index)
   }
 }
 
-bool CRendererBase::CreateIntermediateTarget(unsigned width, unsigned height, bool dynamic)
+bool CRendererBase::CreateIntermediateTarget(unsigned width,
+                                             unsigned height,
+                                             bool dynamic,
+                                             DXGI_FORMAT format)
 {
-  DXGI_FORMAT format = DX::Windowing()->GetBackBuffer().GetFormat();
+  // No format specified by renderer > mirror swap chain's backbuffer format
+  if (format == DXGI_FORMAT_UNKNOWN)
```

Comment:
This setting can be here:

` if (format == DXGI_FORMAT_UNKNOWN || HighPrecisionTexturesSetting == OFF)`